packages/react-native-bpk-component-navigation-bar/src/BpkNavigationBarTextButtonIOS-test.ios.js

```diff
@@ -81,28 +83,59 @@ describe('iOS', () => {
       expect(tree).toMatchSnapshot();
     });
 
-    it('should resepect "tintColor"', () => {
+    it('should render correctly with type="primary"', () => {
       const tree = renderer
         .create(
           <BpkNavigationBarTextButtonIOS
             title="Done"
             leading={false}
             onPress={jest.fn()}
+            type={BUTTON_TYPES.primary}
+          />,
+        )
+        .toJSON();
+      expect(tree).toMatchSnapshot();
+    });
+
+    it('should render correctly with type="primary" and disabled', () => {
+      const tree = renderer
+        .create(
+          <BpkNavigationBarTextButtonIOS
+            title="Done"
+            leading={false}
+            onPress={jest.fn()}
+            type={BUTTON_TYPES.primary}
+            disabled
+          />,
+        )
+        .toJSON();
+      expect(tree).toMatchSnapshot();
+    });
+
+    it('should respect "tintColor"', () => {
+      const tree = renderer
+        .create(
+          <BpkNavigationBarTextButtonIOS
+            title="Done"
+            leading={false}
+            onPress={jest.fn()}
+            type={BUTTON_TYPES.primary}
             tintColor="red"
           />,
         )
         .toJSON();
       expect(tree).toMatchSnapshot();
     });
 
-    it('should resepect "disabledTintColor"', () => {
+    it('should respect "disabledTintColor"', () => {
       const tree = renderer
         .create(
           <BpkNavigationBarTextButtonIOS
             title="Done"
             leading={false}
             onPress={jest.fn()}
             disabledTintColor="red"
+            type={BUTTON_TYPES.primary}
```

Comment:
This is to ensure that `disabledTintColor` takes precedence over any `type` passed in.

Reply:
If you feel this needs to be explained should we include a comment in the source?